README.md

```diff
@@ -185,3 +185,59 @@ sudo: true
 before_script:
   - sudo sysctl -w vm.max_map_count=262144
 ```
+
+**Avoid downloading internet again and again**: When you want to run integration tests with a given plugin like `x-pack` 
```

Comment:
Can you rename the title to `Avoid downloading a plugin from the internet again and again` ?